Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/TestFolderIntegrationTests.cs

```diff
@@ -33,24 +33,28 @@ public void TearDown()
 		[Test]
 		public void GetFolderName()
 		{
-			// Arrange
 			string _workspaceName = $"IntTest_{Guid.NewGuid()}";
 			IServicesMgr servicesManager = testHelper.GetServicesManager();
 			int _workspaceId = WorkspaceHelpers.CreateWorkspace.CreateWorkspaceAsync(_workspaceName,
```

Comment:
create workspace can pontentially fail, so let's move that to try block as well. leave the _workspaceId declaration outside.

Reply:
Done